qa/rpc-tests/getrawtransaction_insight.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python2
+# Copyright (c) 2014-2015 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#
+# Test the new fields added to the output of getrawtransaction
+# RPC for the Insight Explorer by the new spentindex
+#
+
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.authproxy import JSONRPCException
+
+from test_framework.util import assert_equal
+from test_framework.util import initialize_chain_clean
+from test_framework.util import start_nodes, stop_nodes, connect_nodes
+from test_framework.util import wait_bitcoinds
+
+from test_framework.script import CScript, OP_HASH160, OP_EQUAL, OP_DUP, OP_EQUALVERIFY
+
+from test_framework.mininode import COIN, CTransaction
+from test_framework.mininode import CTxIn, CTxOut, COutPoint
+
+from binascii import hexlify
```

Comment:
There are a fair amount of unused imports here, which I think will cause the CI to fail. Here are the imports with the unused ones removed:
```
from test_framework.test_framework import BitcoinTestFramework

from test_framework.util import assert_equal
from test_framework.util import initialize_chain_clean
from test_framework.util import start_nodes, stop_nodes, connect_nodes
from test_framework.util import wait_bitcoinds

from test_framework.mininode import COIN
```